src/main/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpoint.java

```diff
@@ -33,19 +33,30 @@
   @Autowired private CaseService caseService;
 
   /**
-   * the GET end point to return the latest valid Non HI Case by UPRN
+   * the GET end point to return latest valid Case which matches the supplied sample attribute
+   * name/value.
    *
-   * @param uprn the UPRN
+   * @param attributeKey - is the name of the field in the sample data to search by.
+   * @param attributeValue - is the value that target case(s) must contain.
    * @return Returned latest Non HI case with valid address for the UPRN
```

Comment:
this javadoc return value description needs updating

Reply:
Updated.